packages/daemon/src/db/index.ts

```diff
@@ -497,6 +497,10 @@ export const updateAddressTablesWithTx = async (
   timestamp: number,
   addressBalanceMap: StringMap<TokenBalanceMap>,
 ): Promise<void> => {
+  if (Object.keys(addressBalanceMap).length === 0) {
+    // No need to do anything here, the transaction most likely has no inputs and outputs
```

Comment:
"most likely"? Is it possible that this is not the case?

Reply:
No.

Removed the comment

Reply:
Did you push it?

Reply:
Ops, I pushed it to the invalid scripts branch, just cherry-picked it, can you check again?